public/scripts/extensions.js

```diff
@@ -38,7 +38,8 @@ export let modules = [];
 let activeExtensions = new Set();
 
 const getApiUrl = () => extension_settings.apiUrl;
-const sortManifests = (a, b) => parseInt(a.loading_order) - parseInt(b.loading_order) || String(a.display_name).localeCompare(String(b.display_name));
+const sortManifestsByOrder = (a, b) => parseInt(a.loading_order) - parseInt(b.loading_order) || String(a.display_name).localeCompare(String(b.display_name));
+const sortManifestsByName = (a, b) => String(a.display_name).localeCompare(String(b.display_name)) || parseInt(a.loading_order) - parseInt(b.loading_order);
```

Comment:
I mean, likely not that relevant for extension, who should hopefully have clean names, but I feel like whenever you do sorting by alphabetically, you should use something that ignores casings and all weird accents.
I have added `compareIgnoreCaseAndAccents` some time ago.

But yeah, just a nitpick, no need to do it here if you don't want it to.